src/views/portals/utils/DashboardRoutesList.vue

```diff
@@ -43,8 +43,10 @@ export default {
     },
   },
 
-  created() {
-    this.routesPromise = c2c.route.getAll({ limit: 5 });
+  created(query = {}) {
+    query.limit = 5;
+    query.qa = 'draft,great';
+    this.routesPromise = c2c.route.getAll(query);
   },
```

Comment:
```suggestion
  created(query = {}) {
    this.routesPromise = c2c.route.getAll({...query , limit: 5, qa: 'draft,great'});
  },
```

or something similar?